### PR TITLE
Added new model for PlayListModel

### PR DESCRIPTION
**Pull request by nskumar73**
Added a new model for PlayListModel

### CODE REVIEW

**Review by adjayanbaeva — APPROVED** [review]

File: Melody/Model/PlayListModel.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+        /// <summary>
+        /// Genere of the music
+        /// </summary>
+        public string Genre { get; set; }
+
+        /// <summary>
+        /// Artist Name
+        /// </summary>
+        public string Aritst { get; set; }
```
Comment (brillan150) [inline Melody/Model/PlayListModel.cs]: Please correct spelling Aritst -> Artist (spelled correctly in comment, misspelled in code)

File: Melody/Model/PlayListModel.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    /// </summary>
+    public sealed class PlayListModel
+    {
+        /// <summary>
+        /// Name of the artist
+        /// </summary>
+       public string Name { get; set; }
+
+        /// <summary>
+        /// Genere of the music
```
Comment (brillan150) [inline Melody/Model/PlayListModel.cs]: Please correct spelling Genere -> Genre (spelled correctly in code, misspelled in comment)

**Review by brillan150 — CHANGES_REQUESTED** [review]
Thanks for the PR! I've never reviewed one before. Looks great with two spelling corrections.

File: Melody/Model/PlayListModel.cs (hunk 1 above)
Comment (nskumar73) [reply Melody/Model/PlayListModel.cs]: fixed

File: Melody/Model/PlayListModel.cs (hunk 2 above)
Comment (nskumar73) [reply Melody/Model/PlayListModel.cs]: fixed

**Review by brillan150 — APPROVED** [review]
Looks good, thanks!

**Pull request merged**